agent/util-scripts/gold/pbench-list-tools/test-12.00.txt

```diff
@@ -0,0 +1,16 @@
++++ Running test-12.00 pbench-list-tools --help
+
+pbench-list-tools: you specified an invalid option
+
+The following options are available (use neither, one, or the other, but not both):
+
+	-n str --name=str, list the perftool groups which str is used
+
+	-g str --group=str, list the tools used in this str
+
+	-o --with-options, list the options with each tool
+--- Finished test-12.00 pbench-list-tools (status=1)
```

Comment:
Seems like `--help` should exit without an error.

Reply:
Fixed.